templates/_linux.yaml

```diff
@@ -38,6 +38,10 @@ objects:
   spec:
     running: false
     template:
+      metadata:
+        labels:
+          kubevirt.io/domain: ${NAME}
```

Comment:
What will happen to VMs created by hand? I would expect this specific label to be added by the modifying webhook in kubevirt when VMI is created.

Reply:
I don't know the answer to your question. I just know without labels in metadata the virtclt expose command won't work for the vmi